packages/communication-react/package.json

```diff
@@ -172,9 +172,15 @@
     "ts-patch": "^3.1.2",
     "type-fest": "^4.14.0",
     "typescript": "5.4.5",
-    "typescript-transform-paths": "^3.4.7"
+    "typescript-transform-paths": "^3.4.7",
+    "synckit": "0.8.8"
   },
   "beachball": {
     "shouldPublish": false
+  },
+  "overrides": {    
```

Comment:
Anything in this package.json gets published to our npm package, can you add to here: https://github.com/Azure/communication-ui-library/blob/8f6ab24a3fb21c14377cb1d17954fca9c2ef34b2/common/scripts/prepare-package-json.js#L15 a delete of this `overrides` section